src/Workspaces/Core/Portable/Workspace/Solution/ProjectDependencyGraph.cs

```diff
@@ -439,6 +720,21 @@ private void ComputeTransitiveReferences(ProjectId project, HashSet<ProjectId> r
             }
         }
 
+        /// <summary>
+        /// Gets the list of projects that directly or transitively depend on this project, if it has already been
```

Comment:
`directly or transitively` seems redundant.  maybe just `transitively` or `directly or indirectly`?

Reply:
'directly or transitively' is the terminology used throughout the type prior to my change.